interfaces/ISovrynBrownie.sol

```diff
@@ -361,11 +369,24 @@ contract ISovrynBrownie is
 
 	function getLockedSOVAddress() external view returns (address);
 
+	function getFeeRebatePercent() external view returns (uint256);
+
 	function getMinReferralsToPayout() external view returns (uint256);
 
 	function getAffiliatesUserReferrer(address user) external view returns (address referrer);
 
 	function getAffiliateRewardsHeld(address referrer) external view returns (uint256);
 
 	function getAffiliateTradingTokenFeePercent() external view returns (uint256 affiliateTradingTokenFeePercent);
+
+	function swapExternal(
+		address sourceToken,
+		address destToken,
+		address receiver,
+		address returnToSender,
+		uint256 sourceTokenAmount,
+		uint256 requiredDestTokenAmount,
+		uint256 minReturn,
+		bytes calldata swapData
+	) external returns (uint256 destTokenAmountReceived, uint256 sourceTokenAmountUsed);
```

Comment:
Shouldn't we add the other two view functions getSwapExpectedReturn() and checkPriceDivergence() of SwapsExternal module here?